backend/src/PostService/PostService.Api/appsettings.Development.json

```diff
@@ -6,6 +6,6 @@
     }
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Host=localhost;Port=5432;Username=postgres;Password=mysecretpasswordfordevelopment;Database=postgres"
+    "DefaultConnection": "Host=postgres-posts;Port=5432;Username=postgres;Password=mysecretpasswordfordevelopment;Database=postgres"
```

Comment:
Using docker and docker compose it is best practice to stick to environment variables, not hard coded configs.

Check indexer implementation in IConfiguration